packages/react-jsx-highcharts/src/components/Tooltip/Tooltip.js

```diff
@@ -11,11 +11,10 @@ const Tooltip = memo(props => {
   const chart = useChart();
   const Highcharts = useHighcharts();
 
-  restProps.enabled = defaultTo(props.enabled, true);
+  restProps.enabled = props.enabled ?? true;
 
   useEffect(() => {
-    const chartObj = chart.object;
-    chartObj.tooltip = new Highcharts.Tooltip(chartObj, {
+    updateTooltip(chart, {
       ...(Highcharts.defaultOptions && Highcharts.defaultOptions.tooltip),
```

Comment:
I know this line isn't part of this PR, but perhaps it should be moved into `updateTooltip` below?

Reply:
I don't think it can be moved there as updateTooltip is called with only modifiedProps too. Then updateTooltip would reset non modifiedprops to defaults.